### ISSUE TITLE
Exclude from nudge emails articles written before the author came to Penn State

### ISSUE THREAD

**Issue by anaelizabethenriquez** [psu-stewardship/researcher-metadata#163]
The OA policy does not apply to articles written before the author came to Penn State, so those articles should be excluded from nudge emails (where possible). The emails should include only publications with publication dates more than 6 months after the author's start date, similar to our decision not to send nudge emails for articles published in the first six months of the policy's existence.

**Comment by EricDurante** [comment]
While RMD doesn't currently have (to my knowledge) consistent access to data such as date of hire for faculty members, we are already attempting to do something like this with the data that we do have (which is organization affiliation metadata taken from Pure). However, the current cutoff is on the date that we think the person started at Penn State rather than six months past that date. So this should be a relatively minor change unless we're somehow able to incorporate better information about hire dates.

**Comment by nmg110** [comment]
I should be able to get start dates from WorkLion/Workday and get them populated into Activity Insight, if that helps. Maybe Alex could take the file I have and put it in a database so it can be pulled every so often for RMD.

**Comment by EricDurante** [comment]
@nmg110 That could possibly help, but would we be able to get that info for people who are not in Activity Insight? Ideally we'd want to have it for everyone who is in RMD somehow.

Something else that we might need to consider - Is it possible that a faculty member might have two or more different start dates? Suppose a researcher worked at Penn State for a while, then left to work somewhere else for a while, and then came back to Penn State for a different position. We'd want to ignore articles that they published between their two stints at Penn State and count the articles that they published during those two periods of Penn State employment (starting six months after the beginning of each), so we'd actually need to know _all_ of the employment start and end dates for each person. That's what the Pure organization data gives us for folks who are in Pure, although I'm not sure how complete/accurate/up-to-date that information is.

**Comment by nmg110** [comment]
Yes, the file would have all academic faculty, but not the College of Medicine, which we could add to the data when we get their accounts to be created from the college. I believe it is the same report they pull from Workday for Pure too.
